apps/api/src/schedulers/index.ts

```diff
@@ -0,0 +1,9 @@
+import citizenSchedules from './citizenScheduler';
+import insightSchedules from './insightScheduler';
+
+const setupScheduler = () => {
+    citizenSchedules();
+    insightSchedules();
```

Comment:
Consider renaming these to something like "setupCitizenSchedules", as the current sounds like retriving the current schedule objects

Reply:
Done